### DISCUSSION TITLE
npx react-native-asset

### DISCUSSION THREAD

**Discussion by LayMui** [reactwg/react-native-releases#67] [Announcements]
After I upgrade from 0.67.5 to 0.71.x 
when I run my iOS app on simulator
I keep getting this screen that says
JSON value 'N' of type NString cannot be converted to NSNumber
<img width="408" alt="Screenshot 2023-04-11 at 8 33 20 AM" src="https://user-images.githubusercontent.com/11436517/233521099-e8945001-a5be-4adf-a17d-2fc81780800c.png">

I have to run npx react-native-asset to get rid of this error

is that a way to avoid running this command npx react-native-asset?

**Comment by cipolleschi** [comment]
Hi @LayMui, could you detail what you did when upgrading?
Did you ran `yarn install` and `bundle exec pod install` to make sure to pull in the newest versions of the dependencies?

Otherwise, could you provide a reproducer, please?

Also, it would be better to post this as an [issue](https://github.com/facebook/react-native/issues) in the React Native repository.

**Comment by kelset** [comment]
closing as this is not the right place for this.
